doc/jsk_perception/nodes/template_matching_detector.rst

```diff
@@ -0,0 +1,93 @@
+TemplateMatchDetector
+=====================
+
+What is this?
+-------------
+
+.. image:: ./images/template_matching_detector.jpg
+
+Detect object by template matching
+
+
+Subscribing Topic
+-----------------
+
+* ``~input`` (``sensor_msgs/Image``)
+
+  Raw image.
+
+* ``~input/info`` (``sensor_msgs/CameraInfo``)
+
+  Camera Info.
+
+
+Publishing Topic
+----------------
+
+* ``~output/rect`` (``jsk_recognition_msgs/RectArray``)
+
+  Rectangles of detected objects
+
+* ``~output/viz`` (``sensor_msgs/Image``)
+
+  Visualized image of detection result.
+
+
+Parameters
+----------
+
+* ``~queue_size`` (Int, default: ``100``)
+
+  Size of queue
+
+* ``~template_filename`` (String, default: ``template.png``)
+
+  Template image's file name
+
+* ``~min_scale`` (Float, default: ``0.4``)
+
+  Min scale of template
+
+* ``~max_scale`` (Float, default: ``1.6``)
+
+  Max scale of template
+
+* ``~resize_template_num`` (Int, default: ``20``)
+
+  the number of resizing template
+
+* ``~matching_threshold``, (Float, default: ``0.7``)
+
+  Threshold of matching result
+
+* ``~nms_threshold``, (Int, default: ``0.5``)
+
+  Threshold of nms
```

Comment:
`Threshold of nms` seems difficult to understand for beginners.
Could you change it to for example `Threshold of non maximum suppression for overlapped rects`?

Reply:
Modified